src/EnergyPlus/DesiccantDehumidifiers.cc

```diff
@@ -2310,7 +2310,7 @@ namespace DesiccantDehumidifiers {
         if (desicDehum.regenFanType_Num != DataHVACGlobals::FanType_SystemModelObject) {
             Fans::SimulateFanComponents(state, desicDehum.RegenFanName, FirstHVACIteration, desicDehum.RegenFanIndex);
         } else {
-            state.dataHVACFan->fanObjs[desicDehum.RegenFanIndex]->simulate(state, _, _, _, _);
+            state.dataHVACFan->fanObjs[desicDehum.RegenFanIndex]->simulate(state, _, _);
```

Comment:
Why are blank optional arguments passed?

Reply:
Not sure, but I'm glad two of them are gone now.